.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish
+
+on:
+  workflow_run:
+    workflows: [Python]
+    types: [completed]
+    branches:
+      - "v[0-9]+.[0-9]+.x"
+  
+jobs:
+  copy-docs:
```

Comment:
@lionsardesai Should we use a different job ID value?

Reply:
eeeks good catch i only looked at the name key. changing this.